### PR TITLE
correctly export .Stream

### PR DESCRIPTION
**Pull request by calvinmetcalf**
fix for #143

### CODE REVIEW

File: readable.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
-(function (){
+var Stream = (function (){
   try {
-    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
+    return require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
   } catch(_){}
 }());
 exports = module.exports = require('./lib/_stream_readable.js');
+exports.Stream = Stream || exports;
```
Comment (dcousens) [inline readable.js]: `exports.Stream = exports`

So, you could have `exports.Stream.Stream.Stream.Stream === exports`, wait what?

Comment (calvinmetcalf) [inline readable.js]: it's exactly the same situation you have currently with exports.Readable on the next line

Comment (dcousens) [inline readable.js]: All good,  just struck me as odd at first, but you're indeed right.  This is exactly mimicking the node export.

**Comment by dcousens** [comment]
ACK LGTM.

**Pull request merged**